design/BulkAPI.md

```diff
@@ -75,6 +78,11 @@ progress of the job.
 
 - **experiment_types:** Specifies the type(s) of experiments to run, e.g., `"container"` or `"namespace"`.
 
+- **webhook:** The `webhook` parameter allows the system to notify an external service or consumer about the completion status of
+  an experiment-processing job. When a job is completed, this webhook will be triggered to send an HTTP request to a
+  predefined URL, configured either via an environment variable (`webhookURL`) or within a `kruize` configuration
```

Comment:
Is webhookURL still relevant? Looks like the doc needs to be updated

Reply:
Not relevant. Thanks, Dino, for pointing this out. It's been corrected now.